tests/unit/tests_high_five_base.cpp

```diff
@@ -693,6 +693,20 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(readWriteVector2D, T, numerical_test_types) {
     readWriteVector2DTest<T>();
 }
 
+BOOST_AUTO_TEST_CASE(datasetOffset) {
+    std::string filename = "datasetOffset.h5";
+    std::string dsetname = "dset";
+    const size_t size_dataset = 20;
+
+    File file(filename, File::ReadWrite | File::Create | File::Truncate);
+    std::vector<int> data(size_dataset);
+    std::iota(data.begin(), data.end(), 0);
+    DataSet ds = file.createDataSet<int>(dsetname, DataSpace::From(data));
+    ds.write(data);
+    DataSet ds_read = file.getDataSet(dsetname);
+    BOOST_CHECK_EQUAL(ds_read.getOffset(), 2144);
```

Comment:
The 2144 value here is guarantee by the spec somewhere or it is what work on your machine ? Is this portable ?

Reply:
You're right, I didn't consider that this might be different on different machines. 
Thanks for the comments. I'm going with your suggestions of testing `>0`.